### PR TITLE
Add script to check SDL resends OnSeekMediaClockTimer for two apps

### PR DESCRIPTION
**Pull request by OVikhrov**
@aarotar please review

### CODE REVIEW

File: test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_For_Two_Apps.lua [hunk 1]
```diff
@@ -0,0 +1,111 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_For_Two_Apps.lua]: @OVikhrov, please update Requirement summary with related functional requirement also

File: test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_For_Two_Apps.lua [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+-- [HMI_API] New OnSeekMediaClockTimer notification
+--
+-- Description:
+-- <param name="appID" type="Integer" mandatory="true">
+-- <description>The ID of application that relates to this mediaclock position change..</description>
+--
+-- 1. Used preconditions:
+-- a) First SDL life cycle
+-- b) OnSeekMediaClockTimer notification allowed in preloaded file for default app for all HMI levels
+-- c) Apps successfylly registered and consented
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_For_Two_Apps.lua]: @OVikhrov, please check syntax in script

Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_For_Two_Apps.lua]: Accepted

**Comment by OVikhrov** [comment]
@aarotar Plese review iteration#2

**Pull request merged**